### ISSUE TITLE
Error Method get Intl.DateTimeFormat.prototype.format called on incompatible receiver undefined, with Devexpress datagrid

### ISSUE THREAD

**Issue by aepshtin** [hebcal/hebcal-es6#395]
Devepxress is very poplular datagrid system for Angular, React etc
Since hebcal >=5.0, there is an error
TypeError: Method get Intl.DateTimeFormat.prototype.format called on incompatible receiver undefined
after initlize hebcal

This is the error in the devtools console:
main.ts:11 ERROR TypeError: Method get Intl.DateTimeFormat.prototype.format called on incompatible receiver undefined
    at get format (<anonymous>)
    at format (index.mjs:2029:55285)
    at formatDateTime (date.js:41:12)
    at date.js:27:16
    at Object.format (date.js:207:20)
    at Object.<anonymous> (class.js:18:27)
    at object.<computed> [as format] (dependency_injector.js:29:46)
    at Object.format (format_helper.js:41:20)
    at object.<computed> [as format] (dependency_injector.js:29:46)
    at Object.formatValue (m_utils.js:121:21)

Below a link for very minimal example to see the error.
How to install the example:
1) git clone https://github.com/aepshtin/test-hebcal-dx
2) cd test-hebcal-dx
3) npm i
4) ng s

then open the browser at: http://localhost:4200/

and see the error in the console
Just comment out the Zman initilze and see how it works well

![image](https://github.com/hebcal/hebcal-es6/assets/19984165/294aea7b-2e51-488e-987b-9b68779a987b)

In version 4.5.1 it works well.
But i want to use version >=5.0 becuse of the elevation calculation.
Thanks a lot

**Comment by aepshtin** [comment]
In version 4.5.1 it works well.
But i want to use version >=5.0 becuse  of the elevation calculation.
Thanks a lot

**Comment by mjradwin** [comment]
Thanks for the bug report.

We are having difficulty reproducing the error. Are you sure you have pushed your latest test example to GitHub?

Looking at the `package.json` file in https://github.com/aepshtin/test-hebcal-dx repo, we do not see any dependency on `@hebcal/core`. 

Also, the following line of code does not appear anywhere in the test-hebcal-dx repo

```js
let zman = new Zmanim(Location.lookup("Jerusalem"), new Date(), true);
```

Please double-check your test repo

**Comment by aepshtin** [comment]
I'm very sorry. I've pushed it now. Thanks

**Comment by aepshtin** [comment]
I've noticed that the error occurs even when I merely execute:
"Location.lookup('Jerusalem');"
or
"Location.addLocation('test', new Location(31.7683, 35.2137, true, 'Asia/Jerusalem', 'test')),"
without instantiating a new Zmanim object.

**Comment by aepshtin** [comment]
Or even just:
new HDate();

**Comment by mjradwin** [comment]
A cursory examination of the error message you posted seems to point to the bundled Temporal polyfill. More investigation is needed to determine if there is pollution of global namespace for a format function.

**Comment by mjradwin** [comment]
It looks like there was a very recent release of this. We will try updating the dependency and rebuilding @hebcal/noaa and @hebcal/core packages to see if that fixes the issue

https://github.com/fullcalendar/temporal-polyfill/releases/tag/v0.2.1

**Comment by mjradwin** [comment]
This appears to be fixed in `@hebcal/core` version 5.2.2

You can also safely delete the dependency `@hebcal/noaa` from your `package.json`

**Comment by aepshtin** [comment]
Wow! You are a genius!
Thank you!
